runtime/ops/os/sys_info.rs

```diff
@@ -238,7 +238,7 @@ pub fn mem_info() -> Option<MemInfo> {
       if let Some(line) = line {
         let mem = line.split_whitespace().nth(1);
         let mem = mem.and_then(|v| v.parse::<u64>().ok());
-        mem_info.available = mem.unwrap_or(0);
+        mem_info.available = mem.unwrap_or(0) * 1024;
```

Comment:
According to man pages (https://man7.org/linux/man-pages/man5/proc_meminfo.5.html) the unit of the value is written in the line. Maybe we should parse it instead of using kB blindly?

Reply:
it is always in kB [[1]](https://github.com/libuv/libuv/blob/a5c01d4de3695e9d9da34cfd643b5ff0ba582ea7/src/unix/linux.c#L2060) [[2]](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/proc/meminfo.c#n62)